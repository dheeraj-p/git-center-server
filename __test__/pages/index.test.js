import { shallow } from 'enzyme';
import Index from '../../pages/index';

describe('Index', () => {
  it('should render properly', () => {
    const repositories = [
      {
        access_type: 'public',
        _id: '5e8b2ce54b018d3f2dd78df3',
        name: 'first',
        __v: 0,
      },
      {
        access_type: 'public',
        _id: '5e8b2ce54b018d3f2dd78df3',
        name: 'second',
        __v: 0,
      },
    ];
    const wrapper = shallow(<Index repositories={repositories} />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
