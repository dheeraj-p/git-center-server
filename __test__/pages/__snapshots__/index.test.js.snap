// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render properly 1`] = `
"<PageLayout title=\\"GitCenter\\">
  <div className=\\"container-md pb-3\\">
    <div className=\\"row\\">
      <div className=\\"col-lg align-self-center\\">
        <CreateRepository />
      </div>
      <div className=\\"col-lg\\">
        <AddSSHKey />
      </div>
    </div>
    <div className=\\"row\\">
      <div className=\\"col-12\\">
        <Repositories repositories={{...}} />
      </div>
    </div>
  </div>
</PageLayout>"
`;
